config/initializers/devise.rb

```diff
@@ -0,0 +1,312 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'c782e9537c1453dc1bc2d17140391c8115bbcd79bbb641d3f7b544a1a2b6181eed174d1d9a6ecdb6c93ed52e415a0e9229ddd9a66a5aab6ad0e978d183525b46' # rubocop:disable Layout/LineLength
```

Comment:
インラインでの無視設定は避けたほうがよいです。なぜなら右にスクロールしないと RuboCop のコメントが見えないからです。

```suggestion
  # rubocop:disable Layout/LineLength
  # config.secret_key = 'c782e9537c1453dc1bc2d17140391c8115bbcd79bbb641d3f7b544a1a2b6181eed174d1d9a6ecdb6c93ed52e415a0e9229ddd9a66a5aab6ad0e978d183525b46'
  # rubocop:enable Layout/LineLength
```